src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,57 @@
+package lotto.controller;
+
+import lotto.model.Lotto;
+import lotto.model.LottoService;
+import lotto.model.StringParser;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.List;
+import java.util.Map;
+
+public class LottoController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final StringParser stringParser;
+    private final LottoService lottoService;
+
+    public LottoController(InputView inputView, OutputView outputView, StringParser stringParser, LottoService lottoService) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.stringParser = stringParser;
+        this.lottoService = lottoService;
+    }
+
+    public void Game() {
```

Comment:
```suggestion
    public void game() {
```
개발 진행하다보니 놓치신 부분인거 같긴한데 메서드 이름 규칙을 지키지 못한 메서드가 있어 보입니다!